docs/getting-started/upgrade-to-v4.md

```diff
@@ -311,6 +311,7 @@ The way we save data with adapters have slightly changed. With the new Adapter A
 - `provider_account_id`/`providerAccountId` on Account is consistently named `providerAccountId`
 - `access_token_expires`/`accessTokenExpires` on Account renamed to `expires_in`
 - New fields on Account: `expires_at`, `token_type`, `scope`, `id_token`, `session_state`
+- `verificationTokens` collection has been renamed to `verification_tokens` (MongoDB)
```

Comment:
it was called `verification_requests` before. See https://github.com/nextauthjs/adapters/blob/main/packages/typeorm-legacy/src/models/verification-request.js

Reply:
I didn't understand the issue then 😅 , should it then be?
```
- `verification_requests` collection has been renamed to `verification_tokens` (MongoDB)
```

Reply:
Not specific for MongoDB, as the MongoDB adapter hasn't even existed before v4. In previous adapters, the table name was referred to `verification_requests`, because the corresponding adapter methods had too much responsibility. In addition to saving reading from the database, they were also sending the e-mail with the token to the user. this has been moved to the core since then.

Reply:
How should I write this then?
```
- `verification_requests` table has been renamed to `verification_tokens`
```

Reply:
That sounds good.